index.bs

```diff
@@ -889,6 +792,102 @@ user consent to a specific transaction. The structure of these signatures is def
     [[#op-get-assertion]].
 </div>
 
+## Parameters for Credential Creation (dictionary <dfn dictionary>MakeCredentialOptions</dfn>) ## {#dictionary-makecredentialoptions}
+
+<xmp class="idl">
+    dictionary MakeCredentialOptions {
+        // Relying Party / User Metadata:
+        required ScopedCredentialEntity rp;
+        required ScopedCredentialEntity user;
+
+        // Attestation configuration:
+        required BufferSource                         challenge;
+        required sequence<ScopedCredentialParameters> parameters;
+
+        // Optional constraints:
+        unsigned long                        timeout;
+        sequence<ScopedCredentialDescriptor> exclude;
+        Attachment                           attachment;
+        AuthenticationExtensions             extensions;
+    };
+</xmp>
+<div dfn-type="dict-member" dfn-for="MakeCredentialOptions">
+    :   <dfn>rp</dfn>
+    ::  This member contains data about the [=relying party=] responsible for the request.
+
+        Its value's {{ScopedCredentialEntity/name}} member is required, and contains the friendly name of the relying party
+        (e.g. "Acme Corporation", "Widgets, Inc.", or "Awesome Site".
+
+        Its value's {{ScopedCredentialEntity/id}} member specifies the [=relying party identifier=] with which the credential
+        should be associated. If this identifier is not explicitly set, it will default to the [=ASCII serialization of an
+        origin|ASCII serialization=] of the {{WebAuthentication}} object's [=relevant settings object=]'s [=environment settings
+        object/origin=].
+
+    :   <dfn>user</dfn>
+    ::  This member contains data about the user account for which the [=relying party=] is requesting attestation.
+
+        Its value's {{ScopedCredentialEntity/name}} member is required, and contains a friendly name for the user account (e.g.
+        "john.p.smith@example.com", or "John P. Smith").
+
+        Its value's {{ScopedCredentialEntity/id}} member is required, and contains an identifier for the account, specified by
+        the [=relying party=]. This is not meant to be displayed to the user, but is used by the relying party to control the
+        number of credentials - an authenticator will never contain more than one credential for a given relying party under the
+        same {{ScopedCredentialEntity/id}}.
+
+    :   <dfn>challenge</dfn>
+    ::  This member contains a channelge intended to be used for generating the newly created credential's [=attestation object=].
```

Comment:
s/channelge/challenge/ :)

need to expand the explanation (but not necessarily now) -- challenge is used in both regstn and authn protocol runs and has security implications. have submitted https://github.com/w3c/webauthn/issues/404